api3-whitepaper.tex

```diff
@@ -784,44 +784,44 @@ \subsection{dAPI Insurance}
 
 It has been shown that insurance usage not only correlates with macroeconomic growth, but is also a cause of it~\cite{outreville:2013}.
 Nevertheless, insurance is sorely underutilized in the blockchain space.
-One of the main reasons of this is that insurance naturally requires a third party to resolve insurance disputes, and using a mutually-trusted third party for this purpose is against the ethos of decentralization.
-However, the emergence of Kleros~\cite{kleros:2019}, a general purpose on-chain dispute resolution protocol, allows trustless insurance products to be built.
+One of the main reasons of this is that insurance naturally requires a centralized third party to resolve insurance disputes, and using a mutually-trusted third party for this purpose is against the ethos of decentralization.
```

Comment:
The third party isn't required to be centalized and Kleros is an example of a decentralized third party, so I don't get this addition

Reply:
My thought was that true insurance (i.e. provided by a properly licensed and collateralized regulated entity) is necessarily centralized, but this addition is unnecessary so I'll take it out

Reply:
Ah okay, makes sense. I think this won't be clear to the reader unless we give the context of the technicality on which we're not calling this insurance so better to leave it out.